docs/docs/07-examples/01-variable-declaration.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Variable declaration
+id: variable-declaration
+keywords: [Wing example]
+---
+
+### Assignment
+
+```ts 
+let x = 12;
+x = 77; // ERROR: x is non reassignable
+```
+
+```ts
+let var y = "hello";
+y = "world"; // OK (y is reassignable)
+```
```

Comment:
```suggestion
```ts playground
let x = 12;
x = 77; // ERROR: x is non reassignable
```

```ts playground
let var y = "hello";
y = "world"; // OK (y is reassignable)
```
```